### PR TITLE
DEV-43996 Grafana upgrade - Add logzioProvider

### PR DESCRIPTION
**Pull request by copyhold**
window.logzio is filled by logzio iframe product loader.
But what... the obejct came empty. So as a hack add a settimeout. But we know it is wrong.

Migrate logz changes in the userProfile screen - show only basic fields (without social and teams)

### CODE REVIEW

File: public/views/index.html [hunk 1]
```diff
@@ -334,6 +348,33 @@
...
+          pathname +
+          window.location.search;
+        throw new Error('Using grafana not as an iframe, going back to Logzio Web App.');
+      }
+
+      // Set logzio presets on the window
+      window.logzio = window.frameElement.logzioPresets;
+
+      if (window.parent && window.parent.__logzio__) {
+        // window.parent.__logzio__.services.overrideFetch(window);
```
Comment (copyhold) [inline public/views/index.html:369]: leaving this active breaks the app

File: public/views/index.html [hunk 2]
```diff
@@ -334,6 +348,33 @@
...
+        window.parent.__logzio__.services.overrideGrafanaFetch(window);
+      }
+
+      if (!window.logzio) {
+        window.logzio = { productLoadedFlag: true };
```
Comment (copyhold) [inline public/views/index.html:374]: this does not work

File: public/app/AppWrapper.tsx [hunk 3]
```diff
@@ -50,6 +51,16 @@ export class AppWrapper extends React.Component<AppWrapperProps, AppWrapperState
   async componentDidMount() {
     await loadAndInitAngularIfEnabled();
     this.setState({ ready: true });
+
+    // LOGZ.IO GRAFANA CHANGE :: Let app know that grafana loaded successfully
+    setTimeout(() => {
```
Comment (copyhold) [inline public/app/AppWrapper.tsx:56]: need to solve this somehow. setTimeout is a key to flakies

File: public/app/AppWrapper.tsx [hunk 4]
```diff
@@ -4,6 +4,7 @@ import { Provider } from 'react-redux';
 import { Router, Redirect, Switch, RouteComponentProps } from 'react-router-dom';
 import { CompatRouter, CompatRoute } from 'react-router-dom-v5-compat';
 
+// import { productLoaded  } from '@grafana/data';
```
[the remark this answers is not in the document]
Comment (ifatregev) [reply public/app/AppWrapper.tsx:7]: delete

File: public/app/AppWrapper.tsx (hunk 3 above)
Comment (ifatregev) [reply public/app/AppWrapper.tsx:57]: retry better

File: public/app/features/profile/UserProfileEditPage.tsx [hunk 5]
```diff
@@ -112,15 +112,18 @@ export function UserProfileEditPage({
     })),
   ];
 
+  // LOGZIO CHANGES - show limited data in user profile
```
Comment (ifatregev) [inline public/app/features/profile/UserProfileEditPage.tsx:115]: can u add this point to ticket discerption + pr discerption

File: public/views/index.html [hunk 6]
```diff
@@ -273,6 +273,20 @@
         }
       }
 
+      // LOGZ.IO GRAFANA CHANGE :: DEV-19549 Use browser's origin to avoid CORS
```
Comment (ifatregev) [inline public/views/index.html:276]: can u add this point to ticket discerption + pr discerption

File: public/views/index.html (hunk 1 above)
Comment (ifatregev) [reply public/views/index.html:369]: we need this part to add headers, 
you can add a ticket to investigate later, describe the issue, or invest time to investigate.

File: public/views/index.html (hunk 2 above)
Comment (ifatregev) [reply public/views/index.html:374]: delete this, you can add these lines to the investigation ticket if it will help you in the future...

**Review by ifatregev — APPROVED** [review]
great!! minor comments to keep the tickets + PR description informative. 
we'll investigate the 2 tech issues in the next tickets.

File: public/app/AppWrapper.tsx (hunk 3 above)
Comment (vicmiletsky) [reply public/app/AppWrapper.tsx:57]: yes retry should reduce flakiness

File: public/app/features/profile/UserProfileEditPage.tsx (hunk 5 above)
Comment (vicmiletsky) [reply public/app/features/profile/UserProfileEditPage.tsx:115]: We probably should follow same format for these comments so we don't miss them later `// LOGZ.IO GRAFANA CHANGE ::`

**Pull request merged**